webrtc-stats.html

```diff
@@ -1836,7 +1838,10 @@
               </dt>
               <dd>
                 <p>
-                  Represents the total number of connectivity check retransmissions received.
+                  Represents the total number of connectivity check request retransmissions
```

Comment:
I'd suggest counting both requests and responses here. If there's a need to differentiate retransmitted requests and responses, this could be split into two fields, but I can't think of such a need.